problems/countNumbers.js

```diff
@@ -12,9 +13,40 @@
  * returns { 99: 2, 11: 1, 12: 1, 13: 1, 58: 1 }
  */
 
-function countNumbers() {
```

Comment:
This function should take in a parameter as indicated on line 5. So, we need to be mindful to include the parameter in the parenthesis of the function.